angr/engines/pcode/lifter.py

```diff
@@ -0,0 +1,1506 @@
+# FIXME:
+#     - Eliminate Vex references where possible
+#     - Consider moving pieces of lifter classes to higher abstraction layer
+#       to reduce duplication with Vex
+#     - Fix default_exit_target
+#     - Fix/remove NotImplementedError's
+
+from collections import defaultdict
+import pypcode
+import cle
+from archinfo import ArchARM
+from cachetools import LRUCache
+import logging
+import os
+import sys
+
+from.behavior import BehaviorFactory
```

Comment:
`from .behavior import BehaviorFactory`

You missed a space :)

Reply:
Thanks

Reply:
Fixed